Bio/SeqIO/Interfaces.py

```diff
@@ -175,12 +175,20 @@ def __init__(self, handle):
         self._footer_written = False
 
     def write_header(self):
+        """Set _header_written to True.
```

Comment:
This is not helpful. How about:

```
    """Default method for writing file footer.

    If your file format defines a footer, you should implement this method
    in order to write the footer after all the records.

    The default implementation checks the private attribute ._header_written
    to ensure the footer is only written once.
    """"

Reply:
'Default' at the begin of the DocString returns "D401 First line should be in imperative mood; try rephrasing", So I'll try rephrasing this.